ts-typeorm/typeorm/logic.ts

```diff
@@ -10,11 +10,12 @@ export const initialize = async (): Promise<DataSource> => {
    * }
    */ 
     const AppDataSource = new DataSource({
-        type: "postgres",
-        entities: [User],
-        synchronize: true,
-        logging: false,
-    })
+     type: "sqlite",
```

Comment:
This change seems unrelated, but I don't mind it

Reply:
true. like I mentioned in response to @ewucc, I made the change so I (and presumably anyone else who wants to run that sample) can run the sample without setting up postgres locally.